mqtt.kafka.broker/zilla.yaml

```diff
@@ -28,12 +37,45 @@ bindings:
       sni:
         - localhost
     exit: north_mqtt_server
+  north_wss_tls_server:
+    type: tls
+    kind: server
+    vault: my_servers
+    options:
+      keys:
+        - localhost
+      sni:
+        - localhost
+    exit: north_ws_http_server
+
+  # WebSocket server
+  north_ws_http_server:
```

Comment:
This should just be `north_http_server`, with an additional
```
upgrade: websocket
```
for each route below to exit `north_ws_server`.

Reply:
Is `upgrade` a zilla property or a header? I can't find an example it in the docs or the other `ws` examples.

Reply:
`upgrade` is an http header used by WebSocket handshake.

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Protocol_upgrade_mechanism#upgrading_to_a_websocket_connection